students/rob_sanchez/Lesson 2/Fizz_Buzz.py

```diff
@@ -0,0 +1,16 @@
+def FizzBuzz():
+    #variables
+    num1=3
+    num2=5
+    range_start=1
+    range_end=101
+
+    for i in xrange(range_start,range_end):
```

Comment:
This should use the built-in Python3 `range` function rather than the Python2`xrange` function. Please make sure that you are using Python 3 for your class work, rather than Python 2 which comes pre-installed on many personal computers. There are instructions for installing the latest version of Python in the course materials [here](https://uwpce-pythoncert.github.io/PythonCertDevel/supplemental/installing/index.html#installing-python).